modules/gapi/src/backends/cpu/gcpustereo.cpp

```diff
@@ -0,0 +1,50 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#include <opencv2/gapi/stereo.hpp>
+#include <opencv2/gapi/cpu/stereo.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+
+#include <opencv2/calib3d.hpp>
+
+GAPI_OCV_KERNEL_ST(GCPUStereo, cv::gapi::calib3d::GStereo, cv::gapi::calib3d::StereoSetup)
+{
+    static void setup(const cv::GMatDesc&, const cv::GMatDesc&,
+                      const cv::gapi::StereoOutputFormat, const int&,
+                      std::shared_ptr<cv::gapi::calib3d::StereoSetup> &stereoSetup,
+                      const cv::GCompileArgs &compileArgs) {
+        auto stereoInit = cv::gapi::getCompileArg<cv::gapi::StereoInitParam>(compileArgs)
+            .value_or(cv::gapi::StereoInitParam{});
```

Comment:
Why do we make it that configurable? What is the expected scenario?

Reply:
We have an approved API and we also need to configure some kernel parameters 
I think its the best way to solve this case 

I`ll discuss this with @AsyaPronina and @rgarnov